### ISSUE TITLE
[WIP] Move videos from "vimeo" and "youtube" to "bilibili".

### ISSUE THREAD

**Issue by Becavalier** [FreeCodeCampChina/freecodecamp.cn#530]

**Comment by S1ngS1ng** [comment]
Needs discussion. Videos on other platforms may subject to copyright.

**Comment by Becavalier** [comment]
Actually, we have got the oral permission from Quincy that we could move all those videos from "vimeo" to "Bilibili", maybe it's better to confirm again through email?

**Comment by S1ngS1ng** [comment]
True. To clarify, what we DO have permission are the videos they created.

But for this: https://github.com/freeCodeCamp/curriculum/blob/a1a2c3a777ca94ce2a577947c62d88782709ab84/challenges/08-coding-interview-prep/take-home-projects.json#L390 , it's a different story. freeCodeCamp team does not own the copyright to this video either.

Still, linking to this video is always welcome. Though, I am skeptical about downloading and uploading to other platforms. IMHO, this is not what we are supposed to do.

@wudifeixue thoughts?

**Comment by wudifeixue** [comment]
I really like the idea we put videos on Bilibili, it will be a lot faster to load and at the same time not restricted by GFW.  At the same time, I think we should make our own Chinese videos. It just makes sense this way since our whole idea is to have a Chinese community that help people to learn how to program, English video won't help us achieve that goal easily. We can start from the project videos first since they are fairly short and easy to record.

**Comment by wudifeixue** [comment]
For other educational videos, the longer ones I think we can sub them, but then we will have to ask for permission first.

**Comment by S1ngS1ng** [comment]
Agreed, making our own version of the video while including the original link seeks more promising and plausible.
